server/routes/user.js

```diff
@@ -0,0 +1,158 @@
+import { Router } from 'express'
+import ensureAuthenticated from '../utils/passport/ensure-authenticated'
+
+const router = Router()
+
+router
+  .route(`/api/v1/users/:uid/site-subjects`)
+  .get(ensureAuthenticated, async (req, res) => {
+    try {
+      const { prisma } = req.app.locals
+      const { siteSubjects } = await prisma.users.findUnique({
+        where: { uid: req.params.uid },
+        select: { siteSubjects: true },
+      })
+
+      return res.status(200).json({ data: siteSubjects })
+    } catch (error) {
+      console.error(error.stack)
+
+      return res.status(500).send({ message: error.message })
+    }
+  })
+
+router
+  .route('/api/v1/user/:uid/site-subjects/star')
+  .post(ensureAuthenticated, async (req, res) => {
+    try {
+      const { prisma } = req.app.locals
+      const user = await prisma.users.findUnique({
+        where: { uid: req.params.uid },
+        select: {
+          siteSubjects: true,
+        },
+      })
+      const { siteSubjects } = user
+      const siteSubjectSiteIndex = siteSubjects.findIndex(
```

Comment:
can we use `find` instead of `findIndex`?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/241/commits/bf0423c6c7f4d2b24d8c720cb39b4accdaccbbc7

Reply:
that commit doesn't seem to be relevant to this comment.